aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -0,0 +1,134 @@
+    | **Sign-up or sign-in policy** | `B2C_1_SiUpIn`                                    |
+    | **Reset password policy**     | `B2C_1_SSPR`                                      |
+    | **Edit profile policy**       | *\<leave blank\>*                                 |
+
+    Select the **Copy** link next to **Reply URI** to copy the Reply URI to the clipboard. Select **OK** to close the **Change Authentication** dialog, and then select **OK** to create the web app.
+
+## Finish the B2C app registration
+
+Return to the browser window with the B2C app properties still open. Change the temporary **Reply URL** specified earlier to the value copied from Visual Studio, and then select **Save** at the top of the window.
+
+> [!TIP]
+> If you didn't copy the Reply URL, use the SSL address from the Debug tab in the web project properties, and append the **CallbackPath** value from *appsettings.json*.
+
+## Configure policies
+
+Use the steps in the Azure AD B2C documentation to [create a sign-up or sign-in policy](/azure/active-directory-b2c/active-directory-b2c-reference-policies#create-a-sign-up-or-sign-in-policy), and then [create a password reset policy](/azure/active-directory-b2c/active-directory-b2c-reference-policies#create-a-password-reset-policy). Use the example values provided in the documentation for **Identity providers**, **Sign-up attributes**, and **Application claims**. Using the **Run now** button to test the policies as described in the documentation is optional.
+
+> [!WARNING]
+> Ensure the policy names are exactly as described in the documentation, as those policies were used in the **Change Authentication** dialog in Visual Studio. The policy names can be verified in *appsettings.json*.
+
+## Run the app
+
+In Visual Studio, press **F5** to build and run the app. After the web app launches, select **Sign in**.
+
+![Sign into the app](./azure-ad-b2c/_static/signin.png)
+
+The browser redirects to the Azure AD B2C tenant. Sign in with an existing account (if one was created testing the policies) or select **Sign up now** to create a new account. The **Forgot your password?** link is used to reset a forgotten password.
+
+![Azure AD B2C login](./azure-ad-b2c/_static/b2csts.png)
+
+After successfully signing in, the browser redirects to the web app.
+
+![Success](./azure-ad-b2c/_static/success.png)
+
+## Next steps
+
+In this tutorial, you will learned how to:
+
+> [!div class="checklist"]
+> * Create an Azure Active Directory B2C tenant
+> * Register an app in Azure AD B2C
+> * Use Visual Studio to create an ASP.NET Core Web Application configured to use the Azure AD B2C tenant for authentication
+> * Configure policies controlling the behavior of the Azure AD B2C tenant
+
+Now that the ASP.NET Core app is configured to use Azure Active Directory B2C for authentication, the [Authorize attribute](xref:security/authorization/simple) can be used to secure your app.  Continue developing your app by learning to:
+
```

Comment:
The items in this list complete sentences, so a period at the end of each :point_down:.

Reply:
In all seriousness...  I pride myself on grammar nazism, yet I had COMPLETELY forgotten that rule!  Thank you so much for reminding me!